### ISSUE TITLE
Crash: `NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getPackageName()'`

### ISSUE THREAD

**Issue by InfamousUser** [nextcloud/android#8894]
************ CAUSE OF ERROR ************
```java
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getPackageName()' on a null object reference
	at com.owncloud.android.operations.RefreshFolderOperation.sendLocalBroadcast(RefreshFolderOperation.java:733)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:255)
	at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:143)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:235)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:226)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 4 more
```

### Steps to reproduce
1. Setup sync with DAVx5
2. Finish setup
3. Exit out of it

### Expected behaviour
System should return to previous screen.

### Actual behaviour
App shows debug info.

### Environment data
Android version:

Device model: Google Pixel 2

Stock or customized system: stock

Nextcloud app version: 3.17.0

Nextcloud server version: 21.0.4

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by InfamousUser** [comment]
************ CAUSE OF ERROR ************

java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getPackageName()' on a null object reference
	at com.owncloud.android.operations.RefreshFolderOperation.sendLocalBroadcast(RefreshFolderOperation.java:733)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:255)
	at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:143)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:235)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:226)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 4 more

**Comment by joshtrichards** [comment]
Fixed in #9138
